src/test/munit/flow-routing/flow-routing-test-suite.xml

```diff
@@ -50,6 +50,33 @@
     </munit:validation>
   </munit:test>
 
+  <munit:test name="api-with-charset-routing-unsupported-media-type-test">
```

Comment:
I would just call it `unsupported-media-type-test` because the API or charset are indifferent for the purpose of the test.